firmware/screen/src/main.cpp

```
@@ -0,0 +1,400 @@
+      break;
+    case WebsocketsEvent::GotPing:
+      ws.pong();
+      break;
+    case WebsocketsEvent::GotPong:
+      break;
+  }
+  refresh_ui();
+}
+
+static void ws_try_connect() {
+  if (WiFi.status() != WL_CONNECTED) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)